lib/action_subscriber/dsl.rb

```diff
@@ -79,5 +87,15 @@ def routing_key_for(method, routing_key_name)
     def routing_key_names
       @_routing_key_names ||= {}
     end
+
+    def run_action_with_filters(env, action)
+      subscriber_instance = self.new(env)
+      final_block = ->{ subscriber_instance.public_send(action) }
```

Comment:
Stabby procs?

![archer-puts-lana-in-the-danger-zone-gif](https://cloud.githubusercontent.com/assets/207933/5986735/a6ee9676-a8c4-11e4-9907-f97a5e004458.gif)